docs/docsite/rst/community/contributor_path.rst

```diff
@@ -0,0 +1,104 @@
+****************
+Contributor path
+****************
+
+This section describes the contributor journey from the early beginning, through all the stages to becoming a leader who helps shape the future of Ansible. You can use this path as a roadmap for your long-term participation.
+
+Any contribution to the project, even a small one, is very welcome and valuable. Any contribution counts, whether it's feedback on an isssue, pull requiest, or  topic, a documentation change, or a coding contribution. When you contribute regularly, your proficiency and judgment in the related area are increasing fast and, along with this, the importance of your presence in the project.
+
+.. contents::
+    :local:
+
+
+
+Determine your area of interest
+=================================
+
+First, determine areas that are interesting to you, taking into consideration your current experience and experience you'd like to gain. For example, if you use a certain collection, have a look there. See :ref:`how_can_i_help` for more ideas on how to help.
```

Comment:
```suggestion
First, determine areas that are interesting to you, also consider your current experience and what you'd like to gain. For example, if you use a specific collection, have a look there. See :ref:`how_can_i_help` for more ideas on how to help.
```

Reply:
@ugwutotheeshoes thank for the suggestion!
- Let's maybe split the sentence into 2 `First, determine areas that are interesting to you. Consider your current experience and what you'd like to gain.` ?

Reply:
OK, That's better that way.
Sorry about the late reply too.

Reply:
I've updated the suggested edit to match this idea. @ugwutotheeshoes- I think in the future you have the ability to edit your own suggested edits as well. Just click `...` to the top right of the suggestion and it should give you the ability to edit. I'll be doing that here now so I can merge the changes. and thanks!